src/Spryker/Client/OauthPermission/OauthPermission/OauthPermissionReader.php

```diff
@@ -47,12 +47,12 @@ public function getPermissionsFromOauthToken(): PermissionCollectionTransfer
         $request = Request::createFromGlobals();
         $authorizationToken = $request->headers->get(OauthPermissionConfig::HEADER_AUTHORIZATION);
 
-        if (!$authorizationToken) {
+        $accessToken = $this->extractToken($authorizationToken);
```

Comment:
Could this method get null passed now and break? Looks like it.

Reply:
You are fast ;) Yes that's right.

Reply:
Maybe like this? And I should add a test. But I currently have no valilla spryker setup to test this stand alone.